src/app/shared/nav/nav.component.html

```diff
@@ -11,6 +11,11 @@
             <li class="nav-item">
               <a class="nav-link" routerLink="/tasks/create">Create new task</a>
             </li>
+            @if(currentUser.value?.role?.value == "ADMIN") {
```

Comment:
little bit awkward having currentUser.value.role.value value twice 😆

Reply:
Will change when I figure out how to track a local variable in template